chore: remove codeclimate config file

skaldarnar opened the pull request:
I'm not aware that we are actively using [Code Climate](https://codeclimate.com/), and the offering looks to be proprietary anyways.

Therefore, removing the configuration file.

Qwertygiy commented:
5 commits later: I did not, exactly, but it was close enough, I suppose. Next time should be smoother :P

skaldarnar committed:
chore: remove codeclimate config file

I'm not aware that we are actively using [Code Climate](https://codeclimate.com/), and the offering looks to be proprietary anyways.

Therefore, removing the configuration file.

skaldarnar committed:
chore: remove 'b16-tomorrow-dark.css'

I have no idea what this file is about, but it's only reference was in the configuration for code climate.

I assume this is a stale file no longer used, thus removing it.

jdrueckert committed:
Merge branch 'master' into topic/remove-code-climate
